### ISSUE TITLE
cant uplaod VGG.pth on github

### ISSUE THREAD

**Issue by HassenMhenni** [deadvalo/TER#8]
je n'arrive pas à upload le modèle entraîné par vgg sur github, car le fichier dépasse les 100Mo (500Mo), est-ce normal qu'un modèle soit volumineux ?

**Comment by laurentperrinet** [comment]
oui, c'est normal car il contient plusieurs millions de paramètres. Il faut trouver un autre moyen pour le partager - j'utilise le plus souvent nextcloud ou similaire pour garder la synchronisation.

Je suis preneur de toute autre méthode plus moderne :-)
